projectmanagement/src/main/java/life/qbic/projectmanagement/application/AddExperimentToProjectService.java

```diff
@@ -42,11 +45,21 @@ public AddExperimentToProjectService(
   public void addExperimentToProject(ProjectId projectId, String experimentName,
       List<Analyte> analytes, List<Species> species, List<Specimen> specimens) {
     try {
-      requireNonNull(projectId);
-      requireNonNull(experimentName);
-      requireNonNull(analytes);
-      requireNonNull(species);
-      requireNonNull(specimens);
+      requireNonNull(projectId, "project id must not be null during experiment creation");
+      requireNonNull(experimentName, "experiment name must not be null during experiment creation");
+      //Spring CollectionUtils returns true if collection is empty or null
```

Comment:
```suggestion
```
NIT: if you need comments the code should be made more expressive in stead

Reply:
I think its clear from the isEmpty method what happens so no need for a comment I think